docker-compose.appsvc.yml

```diff
@@ -0,0 +1,27 @@
+# Docker Compose (v2)
+
+version: "2"
+services:
+  appsvc:
+    extends:
+      file: docker-compose.base.yml
+      service: web
+    build:
+      context: docker/images/appsvc
+      args:
+        - BASE_IMAGE=${BASE_IMAGE}
+        - http_proxy=${HTTP_PROXY:-}
+        - https_proxy=${HTTP_PROXY:-}
+        - HTTP_PROXY=${HTTP_PROXY:-}
+        - HTTPS_PROXY=${HTTP_PROXY:-}
+        - NO_PROXY=hub,${NO_PROXY:-}
+        - no_proxy=hub,${NO_PR
```

Comment:
This missing some stuff here?

Reply:
@sylus wasn't sure if I needed to include this, but added here and will test shortly today.